include/envoy/event/file_event.h

```diff
@@ -18,11 +18,12 @@ struct FileReadyType {
   static const uint32_t Closed = 0x4;
 };
 
-#define FORCE_LEVEL_EVENTS
 enum class FileTriggerType { Level, Edge, EmulatedEdge };
 
 static constexpr bool optimizeLevelEvents = true;
 
+// For POSIX developers to get the Windows behavior of file events
+// add `#define FORCE_LEVEL_EVENTS` and recompile.
```

Comment:
See if bazel --copt="-DFORCE_LEVEL_EVENTS" works.  Would be good to add that info to the comment.

Reply:
fixed with a comment 342fe7383e53537121d9003b2731bda5640f2da8